From 5f69978142cab4b54becee4583eba3646b1a690c Mon Sep 17 00:00:00 2001
From: xhy <1216993348@qq.com>
Date: Tue, 17 Oct 2017 19:18:38 +0800
Subject: [PATCH 01/11] aaa

---
 Emp.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Emp.java b/Emp.java
index 2d76fd0..e072394 100644
--- a/Emp.java
+++ b/Emp.java
@@ -1,4 +1,4 @@
 public class Emp{
 	private Integer eno;
-	private Integer eno;
+	private Integer En;
 }
\ No newline at end of file
-- 
2.14.2.windows.3

