From 5d3c52b986b7a30a0fc5496c703234989f2fe796 Mon Sep 17 00:00:00 2001
From: xhy <1216993348@qq.com>
Date: Tue, 13 Mar 2018 19:41:12 +0800
Subject: [PATCH 04/11] =?UTF-8?q?=C3=A5=C2=AE=C2=9D=C3=A5=C2=AE=C2=9D?=
 =?UTF-8?q?=C3=A8=C2=A6=C2=81=C3=A6=C2=9B=C2=B4=C3=A6=C2=96=C2=B0?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 Emp.java | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Emp.java b/Emp.java
index 9fe98b7..50e6759 100644
--- a/Emp.java
+++ b/Emp.java
@@ -1,5 +1 @@
-public class Emp{
-	private Integer eno;
-	private Integer En;
-
-	fff}
\ No newline at end of file
+¸üÐÂ´íÎó£¡£¡
\ No newline at end of file
-- 
2.14.2.windows.3

