From dc49de17ec8550037882cd544994b6ea86143d14 Mon Sep 17 00:00:00 2001
From: xhy <1216993348@qq.com>
Date: Tue, 13 Mar 2018 20:12:49 +0800
Subject: [PATCH 09/11] new file

---
 Emp.java | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/Emp.java b/Emp.java
index 40cfa23..0bb739a 100644
--- a/Emp.java
+++ b/Emp.java
@@ -1,8 +1,4 @@
-<<<<<<< HEAD
-git branch --set-upstream-to=origin/xhy master
-eqewe
-why?  
-=======
+
 public class Emp{
 	private Integer eno;
 	private Integer eno;
@@ -10,4 +6,3 @@ public class Emp{
 
 
 }
->>>>>>> 60c0fb27ce0580c2f8d62fdcf2a3646a30a8fc53
-- 
2.14.2.windows.3

