From 97073d42ac6719be3bda5a7ca984f76e4ab5bad0 Mon Sep 17 00:00:00 2001
From: xhy <1216993348@qq.com>
Date: Tue, 13 Mar 2018 20:37:45 +0800
Subject: [PATCH 10/11] patch

---
 Emp.java | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Emp.java b/Emp.java
index 0bb739a..bfa3a50 100644
--- a/Emp.java
+++ b/Emp.java
@@ -3,6 +3,4 @@ public class Emp{
 	private Integer eno;
 	private Integer eno;
 	private String gitpro222222;
-
-
 }
-- 
2.14.2.windows.3

